docs/tutorials/getting_started.ipynb

```diff
@@ -0,0 +1,209 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "source": [
+    "# Getting started\n",
+    "\n",
+    "This tutorial uses safeds on **titanic passenger data** to predict who will survive and who will not, using sex and travel class as features for the prediction.\n"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "1. Load your data into a `Table`, the data is available under `docs/tutorials/data/titanic.csv`:\n"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "outputs": [],
+   "source": [
+    "from safeds.data.tabular.containers import Table\n",
+    "\n",
+    "titanic = Table.from_csv_file(\"data/titanic.csv\")"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "2. Use `OneHotEncoder` to create an encoder, that will be used later to transform the titanic table.\n",
+    "Use the `fit` function of the `OneHotEncoder` to pass the table and the column names, that will be used as features to predict who will survive to the encoder.\n",
+    "The names of the columns before transformation need to be saved, because `OneHotEncoder` will change the names of the fitted `Column`s:\n"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "outputs": [],
+   "source": [
+    "from safeds.data.tabular.transformation import OneHotEncoder\n",
+    "old_column_names = titanic.column_names\n",
+    "encoder = OneHotEncoder().fit(titanic, [\"sex\", \"travel_class\"])"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "3. Transform the table using the fitted encoder, and create a set with the new names of the fitted `Column`s:\n"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "outputs": [],
+   "source": [
+    "transformed_table = encoder.transform(titanic)\n",
+    "new_column_names = transformed_table.column_names\n",
+    "new_columns= set(new_column_names) - set(old_column_names)"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "4. Tag the \"survived\" `Column` and use the new names of the fitted `Column`s as features:"
```

Comment:
i think you could say that `survived` is your target that should be predicted.

Reply:
Done